### PR TITLE
correct patch for sink binding mode in webhook

### PR DESCRIPTION
**Pull request by lberk**

### CODE REVIEW

File: openshift/patches/013-sinkbinding_webhook_corrections.patch [hunk 1]
```diff
@@ -8,6 +8,6 @@ index eebad41b7..84afe81a6 100644
          - name: SINK_BINDING_SELECTION_MODE
 -          value: "exclusion"
 +          value: "inclusion"
- 
-         securityContext:
-           allowPrivilegeEscalation: false
+         - name: POD_NAME
```
Comment (matzew) [inline openshift/patches/013-sinkbinding_webhook_corrections.patch:11]: why do we add/remove these?  

the exclusion/inclusion I get

Comment (lberk) [reply openshift/patches/013-sinkbinding_webhook_corrections.patch:11]: hmmm, you're right it shouldn't :thinking:

Comment (lberk) [reply openshift/patches/013-sinkbinding_webhook_corrections.patch:11]: ah, it's the diff on the patch itself -- the patch diff is still s/exclusion/inclusion

Comment (matzew) [reply openshift/patches/013-sinkbinding_webhook_corrections.patch:11]: OK... :pray:  

/lgtm
/approve

**Pull request merged**